relay-event-normalization/src/normalize/span/description/redis.rs

```diff
@@ -0,0 +1,598 @@
+use aho_corasick::{AhoCorasick, AhoCorasickBuilder, MatchKind};
+use once_cell::sync::Lazy;
+
+/// Returns a redis command if it is a case-insensitive prefix of `seek`.
+pub fn matching_redis_command(seek: &str) -> Option<&str> {
+    let m = REDIS_COMMAND_MATCHER.find(&format!("{seek} "))?;
```

Comment:
The only suboptimal part of this solution is that it adds a string allocation for every search string.

Reply:
This should be possible with two matchers, one with spaces at the end the other one without, if the matcher without spaces matches (after the one without failed) it must be the full command with no trailing data.

Alternative option, you can `find_iter{overlapping}` to get all matches then just select the longest one which has a space or end of input following. That seems pretty straight forward.